src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+import { HeadOfTable } from '../HeadOfTable/HeadOfTable';
+
+export const Navigation: React.FC = () => {
+  return (
+    <>
+      <nav
+        data-cy="nav"
+        className="navbar is-fixed-top has-shadow"
+        role="navigation"
+        aria-label="main navigation"
+      >
+        <div className="container">
+          <div className="navbar-brand">
+            <HeadOfTable />
+          </div>
+        </div>
+      </nav>
```

Comment:
You have `nav` inside `nav`?
Tags in `<HeadOfTable />` and in this component are duplicating